.vib/kubeapps-apis/goss/kubeapps-apis.yaml

```diff
@@ -0,0 +1,23 @@
+commands:
+  check-group-permissions:
+    exec: find /bitnami /opt/bitnami -type d -perm -g+rwX
+    exit-status: 0
+    stdout:
+    {{ range $dirs := .Vars.directories }}
+      {{ range $path := .paths }}
+      - {{ $path }}
+      {{ end }}
+    {{ end }}
```

Comment:
Please remove this section, as we are not setting permissions for any of the `.Vars.directories`.

Also, to check permissions in `vars.yaml`, we could use:
```
directories:
  - mode: "0755" 
    paths:
      - ...
```